stories/documentation/texts/text/text.stories.ts

```diff
@@ -7,13 +7,18 @@ export default {
 } as Meta;
 
 function getTemplate(args: TextBasicStory): string {
-	return `<h1 class="mod-XXXL">Headline</h1>
-<h1>Titre h1</h1>
-<h2>Titre h2</h2>
-<h3>Titre h3</h3>
-<h4>Titre h4</h4>
-<p>Cotton candy shortbread fruitcake jelly candy pie jelly beans. Halvah chocolate bar cheesecake jelly-o chocolate cake macaroon cake bear claw lollipop. Tart candy pastry soufflé candy canes marzipan.</p>
-<p><b>Cotton candy shortbread fruitcake jelly candy pie jelly beans. Halvah chocolate bar cheesecake jelly-o chocolate cake macaroon cake bear claw lollipop. Tart candy pastry soufflé candy canes marzipan.</b></p>`;
+	return `<h1>Titre 1 : Portez ce vieux whisky au juge blond qui fume</h1>
+<h2>Titre 2 : Portez ce vieux whisky au juge blond qui fume</h2>
+<h3>Titre 3 : Portez ce vieux whisky au juge blond qui fume</h3>
+<h4>Titre 4 : Portez ce vieux whisky au juge blond qui fume</h4>
+<h5>Titre 5 : Portez ce vieux whisky au juge blond qui fume</h5>
+<h6>Titre 6 : Portez ce vieux whisky au juge blond qui fume</h6>
+<p>Body 1 : Portez ce vieux whisky au juge blond qui fume</p>
+<p class="u-textS">Body 2 : Portez ce vieux whisky au juge blond qui fume</p>
+<p class="u-textXS">Body 3 : Portez ce vieux whisky au juge blond qui fume</p>
+<p><i>Caption 1 : Portez ce vieux whisky au juge blond qui fume</i></p>
+<p class="u-textS"><i>Caption 2 : Portez ce vieux whisky au juge blond qui fume</i></p>
+<p class="u-textXS"><i>Caption 3 : Portez ce vieux whisky au juge blond qui fume</i></p>`;
```

Comment:
Could we use `.u-fontStyleItalics` instead of the `<i>` element to keep the semantics and style separated?

There's also a `caption` element which could be styled if the the caption styles on Figma are intended to be used as actual captions.

Reply:
Fixed